src/client/index.ts

```diff
@@ -1,12 +1,73 @@
 /**
- * @module Index
+ * OpenFin Notifications provide developers with a uniform way to create, display and organize desktop notifications
+ * as well as responding to notification events.
+ *
+ * Notifications will be displayed as toasts as well as being listed and organized in a Notification Center. The
+ * Notification Center can be accessed by clicking on the system tray notification icon.
+ *
+ * ## Getting Started
+ *
+ * See the [Desktop Services documentation](https://developers.openfin.co/docs/desktop-services) for details on
+ * including a desktop service within your application. Once configured via your application manifest, the API
+ * documented here will function as expected.
+ *
+ * ### Basic example:
+ * ```ts
+ * // Non-Interactive "fire and forget" notification
+ * // Clicking the notification will pass an `notification-activated` event back to the application.
```

Comment:
Updated text, this section still needs completing to give a general high-level overview.

May break that out into a separate story, will also overlap with the website docs.

Reply:
Did this just get removed entirely from `index.ts`?

Reply:
ahh, I'm in the process of moving to a top-level .md file. I removed from index.ts a little early.

Still WIP, but text currently reads as:

> Non-Interactive "fire and forget" notification
> There will be a `notification-closed` event when the service closes the notification (as this occurs in all cases), but it will not be possible to determine
> why the notification was closed.